main.js

```diff
@@ -0,0 +1,46 @@
+//
+// let diceOne = {
+//   val: () => Math.floor((Math.random() * 6) + 1),
+//   roll: () => {
+//     document.querySelector('.d1 div').className = `dice-${diceOne.val()}`
+//   }
+// }
+//
+// let diceTow = {
+//   val: () => Math.floor((Math.random() * 6) + 1),
+//   roll: () => {
+//     document.querySelector('.d2 div').className = `dice-${diceTow.val()}`
+//   }
+// }
+//
+// document.querySelector('.botton').addEventListener('click', () => {
+//   diceOne.roll()
+//   diceTow.roll()
+// })
+
+function getRandom(min, max) {
+   min = Math.ceil(min);
+   max = Math.floor(max);
+   return Math.floor(Math.random() * (max - min + 1)) + min;
+}let dieOne = {
+   value: 10,
+   roll: function() {
+       this.value = getRandom(1, 6);
+       return this.value;
```

Comment:
Well done, it is considered good practice to return a value even from object methods which mutate the object.